### PR TITLE
Add new `direct.quickconnect.cn` domain to Synology dynamic dns service.

### PR DESCRIPTION
**Pull request by ccyen-syno**
- add direct.quickconnect.cn
- sort the list according to https://github.com/publicsuffix/list/wiki/Guidelines#prepare-the-change
- test passed

* [x] Description of Organization
* [x] Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the _PSL txt record in place

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)
  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting

---
__For Private section requests that are submitting entries for domains that match their organization website's primary domain:__

``` 
Seriously, carefully read the downline flow of the PSL and the 
guidelines. Your request could very likely alter the cookie and 
certificate (as well as other) behaviours on your core domain name in 
ways that could be problematic for your business.

Rollback is really not predicatable, as those who use or incorporate 
the PSL do what they do, and when. It is not within the PSL volunteers' 
control to do anything about that.  

The volunteers are busy with new requests, and rollbacks are lowest 
priority, so if something gets broken by your PR, it will potentially 
stay that way for an indefinitely long while.
```
(Link: [about propogation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

Organization Website: https://www.synology.com/
Synology Inc. is a corporation that specializes in Network-attached storage (NAS) appliances.

Reason for PSL Inclusion
====

Hello, I'm engineer from synology.com and we would like to add a new domain for our public dynamic dns service. (refer to https://github.com/publicsuffix/list/pull/62)

DNS Verification via dig
=======

```
dig +short TXT _psl.direct.quickconnect.cn
"https://github.com/publicsuffix/list/pull/1547"
```

make test
=========

the last lines of the command `make test` is as lines in link

https://paste.yunohost.org/ohazazaqec.bash

### CODE REVIEW

File: public_suffix_list.dat [hunk 1]
```diff
@@ -13618,19 +13618,20 @@ syncloud.it
...
+direct.quickconnect.cn
 dsmynas.com
-dsmynas.net
-dsmynas.org
 familyds.com
-familyds.net
-familyds.org
+diskstation.me
+dscloud.me
+dscloud.mobi
```
Comment (sleevi) [inline public_suffix_list.dat:13627]: `.mobi` sorts after `.me`

File: public_suffix_list.dat [hunk 2]
```diff
@@ -13618,19 +13618,20 @@ syncloud.it
 
 // Synology, Inc. : https://www.synology.com/
 // Submitted by Rony Weng <ronyweng@synology.com>
-diskstation.me
 dscloud.biz
-dscloud.me
-dscloud.mobi
+direct.quickconnect.cn
```
Comment (sleevi) [inline public_suffix_list.dat:13622]: Could you explain more about why this name is needed, and the existing domains are insufficient? Effectively, every domain added becomes a cumulative cost to the Internet at large (that ships this list in a wide variety of applications), and so it's useful to understand.

Don't get me wrong, I [don't think it's great](https://github.com/sleevi/psl-problems) that this is the case, but that's the situation today, so it's useful to better understand individual rationale.

Comment (ccyen-syno) [reply public_suffix_list.dat:13622]: Sure.

As `direct.quickconnect.to` is provided as one of our dns service and published in china, we are requested to provide a `.cn` TLD for policy from china administration.

Comment (sleevi) [reply public_suffix_list.dat:13622]: Thanks! For record keeping, do you have a reference to that policy you can point to?

**Comment by dnsguru** [comment]
I went hunting for this rule being mentioned in the rationale, and looked
at CNNIC.  It almost appears that rather than the rule being justification
to add this name, instead adding an ability to offer domains subordinate to
one's registration may actually violate their rules.
https://www.cnnic.com.cn/PublicS/fwzxxgzcfg/201907/t20190726_70776.htm

**Comment by ccyen-syno** [comment]
Hi @dnsguru,

Could you help me point out which rule mentioned in the link was violated so that I can review this carefully.

Thanks for the advice.

File: public_suffix_list.dat (hunk 2 above)
Comment (ccyen-syno) [reply public_suffix_list.dat:13622]: Sure.

If user would like to apply for an ICP filing, a valid domain name is needed ([ref](https://support.huaweicloud.com/intl/en-us/prepare-icp/icp_02_0002.html)). While our original domain direct.quickconnect.to is not a valid domain listed [here](http://xn--eqrt2g.xn--vuq861b/?spm=a2c63.p38356.0.0.3559939axsXaJV#) by MIIT(ministry of industry and information technology), a valid domain is required.

**Comment by dnsguru** [comment]
> Could you help me point out which rule mentioned in the link was violated so that I can review this carefully.

Article 37 of this : [2017 Internet Domain Names](https://www.cnnic.com.cn/PublicS/fwzxxgzcfg/201710/t20171026_69608.htm)

Be certain not to assume that the dynamic DNS service will not considered by MITC to be a 'registry' or 'registrar' in nature, which would make quite a lot of other articles relevent as well.

**Comment by ccyen-syno** [comment]
Our service would provide service like
- User A can register {ID_A}.direct.quickconnect.cn with ip address of **A**'s 
- User B can register {ID_B}.direct.quickconnect.cn with ip address of **B**'s

and A is not able to maliciously direct {ID_A}.direct.quickconnect.cn resolution to ip address of B's

Additionally, what does MITC stand for here?

File: public_suffix_list.dat (hunk 2 above)
Comment (ccyen-syno) [reply public_suffix_list.dat:13622]: Hi @sleevi 

Should I provide any further information for this pull request ?

Comment (sleevi) [reply public_suffix_list.dat:13622]: Thanks! That'll do.

**Pull request merged**